python/kfserving/kfserving/protocol.py

```
@@ -0,0 +1,4 @@
+class ProtocolHandler(object):
+
+    def handleRequest(self,body,model):
+       return "{}"
```

Can we raise an unimplemented error here?